RFCs/FS-1071-witness-passing-quotations.md

```diff
@@ -11,10 +11,15 @@ This RFC covers the detailed proposal for this suggestion and some related desig
 # Summary
 [summary]: #summary
 
-F# quotations of code using SRTP constraint trait calls doesn't carry sufficient information
-to represent the semantic intent of the code, specifically it is missing any record of the
-resolution of SRTP constraints. This RFC addresses this problem by incorporating the necessary
-information into both quotations and the code laid down for dynamic interpretation of quotations.
+F# quotations of code using SRTP constraint trait calls (e.g. using arithmetic) doesn't carry sufficient information
+to represent the full semantic intent of the code. Specifically the data carried in quotations is
+missing any record of the resolution of SRTP constraints. This means that any code interpreting (or compiling)
+from quotations must include 500+ lines of (often buggy) code to re-resolve SRTP constraints.
+This RFC addresses this problem by incorporating the necessary
+information into both quotations and ensuring that assemblies always contain code laid down
+necessary for the dynamic interpretation of quotations.  It also lays the foundation for future
+lifting of the `inline` restriction for generic code using generic arithmetic, and for allowing
+extention methdos to satisfy SRTP constraints.
```

Comment:
Recommended that this is broken into a few bullet points:

* Lays the foundation for future lifting of the `inline` restriction for generic code using generic arithmetic
* Lays the foundation for allowing extension methods to satisfy SRTP constraints

Additionally it would be good to link to the SRTP constraints RFC for readers who want to learn more.